test/support/rake_helpers.rb

```diff
@@ -17,8 +17,7 @@ def test_files
     ar_test_files = ar_test_files.
       split(',').
       map { |file| File.join ARTest::CockroachDB.root_activerecord, file.strip }.
-      then { _1.prepend(COCKROACHDB_TEST_HELPER) unless _1.empty? }.
-      prepend(COCKROACHDB_TEST_HELPER)
+      tap { _1.prepend(COCKROACHDB_TEST_HELPER) unless _1.empty? }
```

Comment:
I forgot that `Kernel#then` returns the result of the block, hence when the condition was false it would return nul rather than an empty array. `Kernel#tap` is what we want: it returns the value of `self`, here the modified array.